src/events/events.ts

```diff
@@ -172,7 +173,16 @@ export const onMemberJoin = async (client: Client, member: GuildMember): Promise
     // The server and logs channel ID of the server that the member joined
     const server: Server = await Server.get(member.guild.id);
 
-    await logTo(client, server, Category.Members, genMemberJoinEmbed(member));
+    try {
+        await logTo(client, server, Category.Members, await genMemberJoinEmbed(member));
+    } catch (e) {
+        await moderationLogsErrorHandler(
+            MemberLogsAction.MemberJoin,
+            member.guild,
+            member.user,
+            e
+        );
+    }
```

Comment:
All `try-catch` blocks for moderation logs have been moved to the `logTo` calls of each moderation log handler.